source/Databricks/source/SqlStatementExecution/Diagnostics/HealthChecks/DatabricksSqlStatementApiHealthRegistration.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading;
+using System.Threading.Tasks;
+using Energinet.DataHub.Core.Databricks.SqlStatementExecution.AppSettings;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using NodaTime;
+
+namespace Energinet.DataHub.Core.Databricks.SqlStatementExecution.Diagnostics.HealthChecks;
+
+public class DatabricksSqlStatementApiHealthRegistration : IHealthCheck
```

Comment:
This is the health check, not a registration.
```suggestion
public class DatabricksSqlStatementApiHealthCheck : IHealthCheck
```